ethereum/core/src/test-support/java/org/hyperledger/besu/ethereum/core/ByteCodeBuilder.java

```diff
@@ -0,0 +1,174 @@
+package org.hyperledger.besu.ethereum.core;
+
+import kotlin.UInt;
+import org.apache.tuweni.bytes.Bytes;
+import org.apache.tuweni.units.bigints.UInt256;
+import org.hyperledger.besu.datatypes.Address;
+
+public class ByteCodeBuilder {
```

Comment:
Where is this class used?

Reply:
It's used in the `TStoreEVMOperationTest` tests to construct the contract bytecode for the EVM-level tests. This makes it more self-documenting and easier to maintain than having long strings of bytecode in the test cases.

Reply:
Please add some javadoc.